src/main/java/fruitshop/strategy/BalanceOperation.java

```diff
@@ -0,0 +1,24 @@
+package fruitshop.strategy;
+
+import fruitshop.model.FruitTransaction;
+import fruitshop.model.Storage;
+
+public class BalanceOperation implements OperationHandler {
+    public static final String READ_FILE_PATH = "src/main/resources/reportToRead.csv";
+    private final Storage storage;
```

Comment:
The `storage` object should not be a field in this strategy class. Each operation should be stateless and only deal with the transaction passed to it. Remove this field and adjust the logic to not rely on a state within the strategy.